core/controllers/ingest.py

```diff
@@ -130,7 +136,7 @@ def parse_body(body: dict) -> tuple[str, int, dict | None, bool]:
 
 def extract_process_validate_tables(
     workbook_data: dict[str, pd.DataFrame], tables_config: dict[str, dict]
-) -> dict[str, Table]:
+) -> dict[str, pd.DataFrame]:
```

Comment:
We need to preserve the `Table` objects because they enable errors found during cross-table validation to be mapped back to the original spreadsheet. Otherwise we'd need to hardcode indexes which I think we should definitely avoid.

Reply:
Noted